packages/react/src/themes/teams/components/Menu/menuItemStyles.ts

```diff
@@ -203,9 +194,17 @@ const menuItemStyles: ComponentSlotStylesInput<MenuItemPropsAndState, MenuVariab
 
         ...(pointing &&
           (vertical
-            ? pointing === 'end'
-              ? { borderRight: `${pxToRem(3)} solid ${v.primaryActiveBorderColor}` }
-              : { borderLeft: `${pxToRem(3)} solid ${v.primaryActiveBorderColor}` }
+            ? !isFromKeyboard && {
```

Comment:
I find this ternary operator difficult to read. Would it be easier to scan if this were flat, like 
```
...(pointing && vertical && !isFromKeyboard && {
    styles here
}),
...(pointing && !vertical && {
    styles here
}),
```

Reply:
Done.